Application.hs

```diff
@@ -30,10 +29,9 @@ import           Network.Wai.Middleware.RequestLogger ( mkRequestLogger, outputF
 import           Network.Wai.Logger                   (clockDateCacher)
 import qualified Network.Wai.Middleware.RequestLogger as RequestLogger
 import           System.Directory
-import           System.Environment                   (lookupEnv)
+import           System.Environment
 import           System.Log.FastLogger                (newStdoutLoggerSet, defaultBufSize, flushLogStr)
 import           System.IO                            (stderr)
-import           System.Posix.Env.ByteString
```

Comment:
My only concern here is performance. I doubt there is any, but there might be some performance implications from switching `ByteString` to `String`.